pkg/operator/controller/garden/garden/components.go

```diff
@@ -775,6 +775,7 @@ func (r *Reconciler) newIstio(ctx context.Context, garden *operatorv1alpha1.Gard
 			{Name: "tcp", Port: 443, TargetPort: intstr.FromInt32(9443)},
 		},
 		false,
+		nil,
```

Comment:
What about the garden cluster case - don't we need to support the same configuration options here?

Reply:
Yes, in general this could also be possible. However, as of now there is currently no support for extensions for gardens. Therefore, I do not know of any potential consumers for this information.

We can add the API to the garden operator to be prepared for the upcoming extension support. Was this what you had in mind?

Reply:
OK, makes sense. As discussed, let's omit this for now. We can always add the API fields later if really needed.